### DISCUSSION TITLE
初學日文50音

### DISCUSSION THREAD

**Discussion by CeBru13** [1101-GS4719-C/Python-Community-GS4719-C#2684] [General]
我在剛學習日文的時候，經歷過背50音的時期，我會一行一行的背，好不容易背起來，結果突然考我還要慢慢あいうえお的數才會，後來老師教我們要跳著順序背才有效，這才改善我的50音，大家可以分享一下你們的方法

**Comment by CeBru13** [comment]
分享一個app叫做maru日文五十音特訓
有免費板可以用，介面做的很簡約可愛，還有懲罰-------看廣告，只有失敗才會看廣告超好笑的

**Comment by christaaah** [comment]
我的話要練習五十音的話我都是從日文歌的歌詞開始的，如果有喜歡的日文歌的話，可以試著不看羅馬拼音而是看平假名和片假名的歌詞，這樣在漸漸熟悉五十音之後還可以順便記單字。順帶一提，我現在五十音的順序完全沒辦法按照順序背完。

**Comment by wcyeee** [comment]
我覺得學語言真的是一件很難的事...我之前嘗試想要學學看日文，但每次開始五十音之後就卡住不想背了哈哈哈 希望用了推薦的應用程式之後可以讓我下次撐得更久！希望可以趕快學會日文！

**Comment by kzx-creat** [comment]
之前修過日文一A，考試拿到考卷後，要先把五十音全部寫在考卷某個角落才開始答題，因為也是要從五十音第一排開始背。但是到了那個學期末，開始可以單獨念出很多個五十音，我想是因為看過很多遍，越來越熟悉的關係。

**Comment by Mminniehuang** [comment]
我一開始也是用MARU日文五十音特訓！除了懲罰之外它還有用深淺表示你的熟練程度，我覺得很有趣也很好用。
現在的話大部分還是跟著老師透過朗讀會話文之類的慢慢熟悉起來，我覺得也是很有效的方法！

**Comment by qiu0109** [comment]
之前上日文一A的時候，老師是教我們兩兩一組，將五十音寫在卡片上然後請隊友丟出卡片而自己要馬上念出卡片上的音，我覺得這個方法還不錯，而且有隊友陪你一起學日文會比較有動力去學習。

**Comment by yue083139** [comment]
感覺要熟記五十音還是要從脫離用平常熟悉的思維來思考做起，可以多用其他方式來記憶，例如聽到就寫出對應五十音或是用奇怪的順序來背，例如橫著背的順序:あかさたなはまやらわ之類的，也可以在平假名和片假名之間進行相互的轉換之類的，會有更深刻的背誦印象而不是單純的短期記憶。

**Comment by ed112601513** [comment]
我學五十音是先把平假名和片假名都背起來，然後隨便找日文文章來讀，不需要會看得懂文章寫什麼，單純測試自己會不會念就好，這樣可以有效的讓自己更加熟悉五十音，但缺點是不常出現的還是會常常忘記

**Comment by Elaine4488** [comment]
我初學的時候也是一定要照順序才會，平假名還可以，反倒是片假名，我花超久時間背的，超容易忘記或搞混。但我覺得就是要多熟悉它，日常生活中看到就會考考自己，看過很多遍，就會越來越熟。

**Comment by SendaiHatsuki** [comment]
我個人是很喜歡Marumaru的唱歌學日文，他的漢字和片假名都會標註平假名的拼音，然後邊聽自己喜歡的日文歌邊看歌詞，很快就可以熟悉50音，和一些比較常用到的漢字發音。

**Comment by XiangLuen** [comment]
我習慣是在youtube聽日文歌，有些歌會有日文和中文字幕，看久了就會懂，接者可以考慮看Todaii japanese，他會給你日本的新聞，然後會唸給你聽，我認為這樣可以學習到很多，不會像考檢定那樣的死板，而且說真的我認為語言不能只會聽跟讀，而是要配上說才行，因為你不可能去日本都要靠翻譯人員。

**Comment by lilipop6405** [comment]
我放棄讀日文，我真的不是文組的料。我上日文一A的時候也是五十音背了很久，尤其是日文單字，根本有背沒有差不多，不然就是要背很久，最後發現我不是讀語文的料@.@ ......

**Comment by yuuu3957** [comment]
我背的方法是每一天都把50音寫一遍，寫個一個禮拜左右就背得差不多了。但這時候還是有點要重頭數。
然後看到新單字或是生活中遇到日文的時候，會嘗試自己先念念看，念多次之後就比較能直接對應。

**Comment by alice20040127** [comment]
我自己是一開始純靠硬記的，但就是感覺成效沒有很好，後來就轉而開始先看一些日文單字，看不懂的時候就回去查它到底是怎麼念的，然後再想起這個單字的時候，就會順帶回想起它裡面的那些音。最近的我是感覺真的要常看日文，有一陣子沒看要考試了打開開始看，突然就發現有些長得相像的音會搞混，像是ぬ和ね之類的。

**Comment by LINSHIHMIN** [comment]
一開始就是直的背（あかさたな這樣），橫的也背（あいうえお這樣），起初先照著順序我覺得沒問題，後來比較進階的時候可以試著讓朋友考考自己，比如隨機唸兩個音，甚至可以三個音、四個音之類的，因為之前上日文課的時候老師都會這樣考試，所以我覺得這樣的訓練是蠻不錯的，最主要還是要多看，多接觸日文，慢慢地看到任何一個音都可以直接知道怎麼念。片假名也是一樣的學習方式，一開始我覺得比較難背尤其是クケタ，我一開始很容易混淆，但隨著背越來越多單字，對片假名也就越熟悉了！
